modules/news/news.model.js

```diff
@@ -0,0 +1,18 @@
+const newsType = `
+_id: ID!
+title: [Language!]
+text: [Language!]
+images: [PrimaryImage!]
+video: String!
+author: Author!
+date: String!`;
+
+const newsInputType = `
+title: [LanguageInput!]
+text: [LanguageInput!]
+images: [PrimaryImageInput!]
+video: String!
+author: AuthorInput!
```

Comment:
it has to be author image field, it is not required but it is necessary to have it

Reply:
fixed